xonsh/platform.py

```diff
@@ -240,7 +246,12 @@ def scandir(path):
                      'usr\\share\\bash-completion\\completions'),
         os.path.join(GIT_FOR_WINDOWS_PATH,
                      'mingw64\\share\\git\\completion\\git-completion.bash'))
-
+    PATH_DEFAULT = tuple(winreg.QueryValueEx(winreg.OpenKey(
+            winreg.HKEY_LOCAL_MACHINE,
+            r'SYSTEM\CurrentControlSet\Control\Session Manager\Environment'),
+        'Path')[0].split(os.pathsep))
```

Comment:
We do inherit from the calling environment, as you say @melund, so the use case is rare (but important).  Suppose that the calling env doesn't have a $PATH, then we should still be able to run programs by having a reasonable fallback. 

For example, to test this, I was doing something like:

```
xonsh level 0 $ echo $PATH
inherited:path:values
xonsh level 0 $ del $PATH
xonsh level 0 $ xonsh
xonsh level 1 $ echo $PATH
default:path:values
```